iree/base/internal/status.h

```diff
@@ -22,6 +22,13 @@
 #include "absl/flags/flag.h"
 #include "absl/strings/string_view.h"
 #include "iree/base/internal/logging.h"
+#include "iree/base/target_platform.h"
+
+// WinGDI.h, included indirectly through absl/flags/flag.h, defines `ERROR`.
+// undef to avoid name conflicts.
+#if defined(IREE_PLATFORM_WINDOWS)
+#undef ERROR
```

Comment:
can change this and the logging thing to just include windows.h - keep the nasty contained

Reply:
Done. Needed to adjust defines a bit to keep it working.

I referenced https://github.com/abseil/abseil-cpp/blob/1948f6f967e34db9793cfa8b4bcbaf370d039fd8/absl/copts/copts.py#L112-L121 and https://github.com/google/xrtl/blob/master/xrtl/port/windows/base/windows.h.